src/fs.zig

```diff
@@ -121,9 +121,8 @@ pub fn serveFile(
 
     //Carrot Return after headers to tell clients where headers end, and body starts
     try stream.writeAll("\r\n");
-    try response.flush();
 
-    const out = response.buffered_writer.unbuffered_writer.context.handle;
+    const out = stream.context.handle;
```

Comment:
With the above 2 changes, this can remain as-is.
I believe buffering is worth it and a quick benchmark showed a gain of 30k req/s.